<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QMainWindow" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>692</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1041</width>
    <height>692</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>安防-报警视频联动管理平台</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/image/main.png</normaloff>:/image/main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;宋体&quot;;</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabData">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>0</height>
          </size>
         </property>
         <attribute name="title">
          <string>警情</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabHost">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>主机</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabMap">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>0</height>
          </size>
         </property>
         <attribute name="title">
          <string>地图</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor" stdset="0">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="iconSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxBottom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnBuFang">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>全部布防(&amp;B)</string>
           </property>
           <property name="icon">
            <iconset resource="main.qrc">
             <normaloff>:/image/bufang.png</normaloff>:/image/bufang.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCheFang">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>全部撤防(&amp;C)</string>
           </property>
           <property name="icon">
            <iconset resource="main.qrc">
             <normaloff>:/image/chefang.png</normaloff>:/image/chefang.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabDefence">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabDefenceMap">
       <attribute name="title">
        <string>防区面板</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QGroupBox" name="boxMain">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDefencePanel">
       <attribute name="title">
        <string>主机面板</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>796</width>
             <height>596</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_config"/>
   <addaction name="action_host"/>
   <addaction name="action_defence"/>
   <addaction name="action_data"/>
   <addaction name="action_exit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">padding:3px 3px 3px 3px;
font: 9pt &quot;宋体&quot;;</string>
   </property>
  </widget>
  <action name="action_config">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/image/config.png</normaloff>:/image/config.png</iconset>
   </property>
   <property name="text">
    <string>【系统设置】</string>
   </property>
   <property name="toolTip">
    <string>系统设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="action_host">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/image/main.png</normaloff>:/image/main.png</iconset>
   </property>
   <property name="text">
    <string>【主机管理】</string>
   </property>
   <property name="toolTip">
    <string>主机管理</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
  <action name="action_defence">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/image/defence.png</normaloff>:/image/defence.png</iconset>
   </property>
   <property name="text">
    <string>【防区管理】</string>
   </property>
   <property name="toolTip">
    <string>防区管理</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="action_data">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/image/select.png</normaloff>:/image/select.png</iconset>
   </property>
   <property name="text">
    <string>【警情查询】</string>
   </property>
   <property name="toolTip">
    <string>警情查询</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/image/close.png</normaloff>:/image/close.png</iconset>
   </property>
   <property name="text">
    <string>【退出】</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
